packages/notebook/src/executionindicator.tsx

```diff
@@ -178,6 +193,10 @@ namespace ExecutionIndicatorComponent {
      * The application language translator.
      */
     translator?: ITranslator;
+    /**
+     * The current notebook.
+     */
+    nb?: Notebook;
```

Comment:
```suggestion
    notebook?: Notebook;
```

Let's avoid unnecessary abbreviations in public interfaces :)